### PR TITLE
Add zone UI info

### PR DESCRIPTION
**Pull request by RosemaryOrchard**
## Proposed change

Add a screenshot of the UI config for a zone, and some info on how to set the location of a zone.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

A user in Discord today was confused by how to configure a zone. This isn't the first time I've seen that, so I thought adding some documentation would be a good first step.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/zone.markdown [hunk 1]
```diff
@@ -16,6 +16,10 @@ Zones allow you to specify certain regions on earth (for now). When a device tra
 
 Zones can be added and managed through the user interface at **{% my zones title="Settings -> Areas & Zones" %}**.
 
+![Screenshot of the UI for adding or editing a zone](/images/integrations/zone/zone_edit_ui.png)
+
+In the user interface you can add a zone by specifying the GPS co-ordinates, or by dragging the icon. You can adjust the radius (except of the Home zone) by adjusting the size of the zone circle.
```
Comment (frenck) [inline source/_integrations/zone.markdown]:
```suggestion
You can add a zone in the user interface by specifying the GPS coordinates or dragging the icon on the map. You can adjust the zone radius (except for the Home zone) by changing the size of the zone circle.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Small readability suggestion.

**Comment by RosemaryOrchard** [comment]
Nice improvement, thank you!

**Review by frenck — APPROVED** [review]
Thanks,  @RosemaryOrchard 👍

**Pull request merged**